physical/couchdb.go

```diff
@@ -160,6 +162,19 @@ func buildCouchDBBackend(conf map[string]string, logger log.Logger) (*CouchDBBac
 		password = conf["password"]
 	}
 
+	maxParStr, ok := conf["max_parallel"]
+	var maxParInt int
+	var err error
+	if ok {
+		maxParInt, err = strconv.Atoi(maxParStr)
+		if err != nil {
+			return nil, errwrap.Wrapf("failed parsing max_parallel parameter: {{err}}", err)
+		}
+		if logger.IsDebug() {
+			logger.Debug("couchdb: max_parallel set", "max_parallel", maxParInt)
+		}
+	}
```

Comment:
There can be, but since we don't call the `sql` function to set the max open connections, it's unnecessary (using 0 will, for the permit pool, use the default anyways).